src/components/Battles.ts

```diff
@@ -189,6 +189,11 @@ export interface IPokemonBattleOptions {
      * Things displayed in the battle.
      */
     things: IBattleThings;
+
+    /**
+     * Audio theme to be played after the battle is done.
+     */
+    endingtheme: string;
```

Comment:
What if the battle shouldn't have a theme played after it finishes (example: super serious cutscene in some fan mod)?

Reply:
Otherwise LGTM

Reply:
It could be part of partial battle options and if they specify "None" for endingtheme audioplayer.stopall would be called after the battle instead of an endingtheme.

Reply:
:( but what if there are animations already playing?

Let's just allow it to be `?: string` (optional), and only do something with it if it's `!== undefined`.